java/ql/automodel/src/AutomodelApplicationModeCharacteristics.qll

```diff
@@ -30,7 +30,9 @@ newtype TApplicationModeEndpoint =
       arg.asExpr() = argExpr and call = argExpr.getCall() and not argExpr.isVararg()
     )
   } or
-  TInstanceArgument(Call call, DataFlow::Node arg) { arg = DataFlow::getInstanceArgument(call) } or
+  TInstanceArgument(Call call, DataFlow::Node arg) {
+    arg = DataFlow::getInstanceArgument(call)
+  } or
```

Comment:
I puzzled a bit over this reformatting. I'm guessing you took the after state (where you have `and not call instanceof ...` -- long enough to change the formatting) and removed the newly added bits, but didn't actually autoformat that version.

Reply:
that's exactly what happened!